Update squads.md page (#1905)
medic/cht-docs: content/en/community/squads.md

  - Test reports
  - Verified software

### 6. 🛋️ Review

> When the squad confirms the solution meets requirements, submit a PR for review and nominate an experienced CHT engineer to lead the technical assessment. After passing both technical review and allowing a few days for community feedback, merge the approved code.

Reviewer: ```suggestion
When the squad confirms that the solution meets the requirements, submit a pull request for review and nominate a CHT maintainer to lead the technical assessment. After passing both technical review and allowing a few days for community feedback, merge the approved code.
```

> **Outputs**: 

Reviewer: ```suggestion

**Outputs**: 
```
